apple/testing/default_runner/macos_test_runner.template.sh

```diff
@@ -160,13 +166,41 @@ xcrun llvm-cov \
   "$test_binary" \
   @"$COVERAGE_MANIFEST" \
   > "$COVERAGE_OUTPUT_FILE" \
-  2> "$error_file" \
-  || llvm_cov_status=$?
+  2> "$export_error_file" \
+  || llvm_cov_export_status=$?
+
+llvm_cov_json_export_status=0
+readonly json_export_error_file="$TEST_TMP_DIR/llvm-cov-json-export-error.txt"
+if [ "$COVERAGE_PRODUCE_JSON" = true ]; then
```

Comment:
In other cases I'm just checking if the variable in question is set at all so folks don't have to worry about specific values, the downside is if folks want to negate it that doesn't work, wdyt?

Reply:
Checking for existence sounds good to me